packages/styles/src/modal/test/modal.html

```diff
@@ -28,7 +28,7 @@ <h1 class="as-title">This is a modal</h1>
         natus voluptatem praesentium itaque sunt commodi odio incidunt fugiat, cupiditate ipsum officiis eligendi?
       </p>
       <hr>
-      <h2 class="as-subheader">I have no padding</h2>
+      <h2 class="as-subheader">I have padding</h2>
```

Comment:
Should we show a real example instead of a lorem ipsum? Try to copy the invision example or something like that